package.xml

```diff
@@ -1,6 +1,6 @@
 <package>
   <name>rqt_robot_steering</name>
-  <version>0.5.8</version>
+  <version>0.5.9</version>
```

Comment:
Please do not change the version number. That will happen later after your PR when the package is actually being released.